patches/gnome-inform7.patch

```diff
@@ -0,0 +1,671 @@
+diff -ru gnome-inform7-6M62/com.inform7.GUI.gschema.xml gnome-inform7-6M62-patched/com.inform7.GUI.gschema.xml
+--- gnome-inform7-6M62/com.inform7.GUI.gschema.xml	2013-09-11 02:19:32.000000000 -0400
++++ gnome-inform7-6M62-patched/com.inform7.GUI.gschema.xml	2019-01-14 18:29:10.202971332 -0500
+@@ -1,30 +1,30 @@
+ <schemalist>
+ 
+-  <enum id="com.inform7.GUI.preferences.font-set-enum">
++  <enum id="com.inform7.IDE.preferences.font-set-enum">
+     <value nick="Standard" value="0" />
+     <value nick="Monospace" value="1" />
+     <value nick="Custom" value="2" />
+   </enum>
+ 
```

Comment:
It would be nice if you could submit this patch upstream.

Reply:
Already merged upstream. I'm only still using the patch here so I could continue using a tall of the most recent release. But future releases won't require a patch like this.